src/main/kotlin/carracing/game/domain/data/Car.kt

```diff
@@ -0,0 +1,13 @@
+package carracing.game.domain.data
+
+private const val INITIAL_POSITION = 0
+
+data class Car(
+    var position: Int = INITIAL_POSITION,
+) {
+    fun move(randomNumber: Int) {
```

Comment:
Does the `Car` really need to know that it's dealing with a random number? What if we abstract the parameter name one level further?